linode/manager
# feat: [M3-7288] - AGLB create page with Actions buttons.

## Description 📝
AGLB create page with Actions buttons.

**Note:** Action buttons behavior, Stepper component contents are not part of this PR scope will be implemented in the followup subtasks.

## Preview 📷
![image](https://github.com/linode/manager/assets/119517080/a80563b6-ba7f-4924-ab62-6d5e7f1d7671)

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Checkout the branch and run the app locally
- Enable MSW


### Verification steps 
(How to verify changes)
- Navigate to http://localhost:3000/loadbalancers
- Click on "Create Load Balancer" Button.
- Verify Load Balancer Create page W.R.T mockup.
- Verify Stepper component functionality by clicking next and previous.

## As an Author I have considered 🤔

*Check all that apply*
